Source/DedicatedServer/Public/ServerConsole.h

```diff
@@ -15,6 +15,24 @@ DECLARE_LOG_CATEGORY_EXTERN( LogServerConsole, Log, All );
 	#elif PLATFORM_LINUX
 		#include "Public/Linux/LinuxConsoleOutputDevice.h"
 		#define FOutputDeviceConsolePlatform FLinuxConsoleOutputDevice
+
+		struct coords
```

Comment:
No, there's just a struct for mouse events. Any code on the web uses simple ints to get the values, but I prefer to have a struct since it's more easy.